scripts/mail-alarm

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Is this part of the black reformatter?

Reply:
No, typically, this is my initial modification when addressing the upgrade ticket, and next, I will run some formatters and linters.